### PR TITLE
Added support for specifying cookies with the `-b` flag

### PR DESCRIPTION
**Pull request by 7047payloads**
I added some code to accept a `key=value` pair for cookies using the `--cookies` flag or the `-b` shorthand. The cookies can still be added using the longer `--header` flag way.

> I've written some code to implement this feature.

_Originally posted by @7047payloads in https://github.com/epi052/feroxbuster/issues/442#issuecomment-1004090323_

### CODE REVIEW

**Comment by epi052** [comment]
Hey there, thanks for taking the time to submit this PR! I'm trying to wrap a few things up, but will review it soon (hopefully this week).  Just wanted to let you know that I see it, thanks again!

File: src/parser.rs [hunk 1]
```diff
@@ -250,6 +250,18 @@ pub fn initialize() -> App<'static, 'static> {
                     "Specify HTTP headers (ex: -H Header:val 'stuff: things')",
                 ),
         )
+        .arg(
+            Arg::with_name("cookies")
+                .short("b")
+                .long("cookies")
+                .value_name("COOKIE")
+                .takes_value(true)
+                .multiple(true)
```
Comment (epi052) [inline src/parser.rs:259]: multiple values are allowed in the parser definition, but only a single `Cookie: stuff=things` makes it into the actual request (the last one to be iterated over in the loop).

A single Cookie is preferred, as it's HTTP/1.1 compliant, so let's remove the `multiple(true)` and make sure documentation shows how to send multiple key/values in a single `--cookies` param.

**Review by epi052 — CHANGES_REQUESTED** [review]
Thanks again for the PR! In addition to the review comments, below are a few items we'll need to sort out in order to get it merged. 

- [ ] bump version to 2.5.0
- [documentation](https://epi052.github.io/feroxbuster-docs/docs/)
  - [ ] update [help output section](https://epi052.github.io/feroxbuster-docs/docs/configuration/command-line/)

**Comment by 7047payloads** [comment]
Hey! I made a couple modifications. I'm keeping the multiple flags config set to true. Each `key=value` pair would be supplied to individual `-b` flags. These cookies are coerced together into a single cookie header, joined with semicolons, HTTP/1.1 compliant now.

**Comment by epi052** [comment]
Thanks for this! I merged your code along with a few minor tweaks and dependency updates. I'm waiting to get another PR or two merged before i actually tag the 2.5.0 release, but it shouldn't be too long, i don't think.

Thanks again!

**Pull request merged**